apps/docs/app/ui/layout/header/Header.tsx

```diff
@@ -1,20 +1,100 @@
 "use client";
 
+import { useContext, useEffect, useState, useRef, type ReactNode } from "react";
```

Comment:
![image](https://github.com/user-attachments/assets/44af89f2-e565-4cd8-9f95-1fe7654119df)
There's a certain viewport where getting started is on two lines